### PR TITLE
iCalendar flow improvements and naming

### PR DESCRIPTION
**Pull request by klaustopher**
Text fixes for the iCal sharing

---

Closes https://community.openproject.org/projects/openproject/work_packages/49247/activity

### CODE REVIEW

File: frontend/src/app/shared/components/modals/get-ical-url-modal/query-get-ical-url.modal.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export class QueryGetIcalUrlModalComponent extends OpModalComponent implements O
       return;
     }
 
-    let icalUrl = '';
+    const icalUrl = '';
```
Comment (aaron-contreras) [inline frontend/src/app/shared/components/modals/get-ical-url-modal/query-get-ical-url.modal.ts]: This seems to be an unused constant. We can remove it and appease eslint :)
```suggestion
```

File: frontend/src/app/shared/components/copy-to-clipboard/copy-to-clipboard.service.ts [hunk 2]
```diff
@@ -40,14 +40,14 @@ export class CopyToClipboardService {
...
-  copy(content:string) {
+  copy(content:string, successMessage?:string) {
     if (!navigator.clipboard) {
       // fallback for browsers that don't support clipboard API at all
       this.addNotification('addError', this.I18n.t('js.clipboard.browser_error', { content }));
     } else {
       void navigator.clipboard.writeText(content)
         .then(() => {
-          this.addNotification('addSuccess', this.I18n.t('js.clipboard.copied_successful'));
+          this.addNotification('addSuccess', successMessage || this.I18n.t('js.clipboard.copied_successful'));
```
Comment (aaron-contreras) [inline frontend/src/app/shared/components/copy-to-clipboard/copy-to-clipboard.service.ts:50]: Very nice 👍

File: config/locales/js-en.yml [hunk 3]
```diff
@@ -392,15 +392,16 @@ en:
...
-      warning: "Please don't share this URL with external users. Anyone with this link will be able to view work package details without an account or password."
-      token_name_label: "Token name"
-      token_name_placeholder: "Type a name for this token, e.g. \"My calendar application\""
-      token_name_description_text: "This name will be used to distinguish it form other tokens in the <a href=\"my/access_token\", target=\"_blank\">access tokens list</a>. It can only be used once per calendar."
+      description: "You can use the URL (iCalendar) to subscribe to this calendar in an external client and view up-to-date work package information from there."
+      warning: "Please don't share this URL with other users. Anyone with this link will be able to view work package details without an account or password."
+      token_name_label: "Where will you be using this?"
+      token_name_placeholder: "Type a name, e.g. \"Phone\""
+      token_name_description_text: "If you subscribe to this calendar from multiple devices, this name will help you to distinguish between them in your <a href=\"my/access_token\", target=\"_blank\">access tokens</a> list."
```
Comment (aaron-contreras) [inline config/locales/js-en.yml]:
```suggestion
      token_name_description_text: "If you subscribe to this calendar from multiple devices, this name will help you distinguish between them in your <a href=\"my/access_token\", target=\"_blank\">access tokens</a> list."
```

File: config/locales/js-en.yml [hunk 4]
```diff
@@ -392,15 +392,16 @@ en:
...
       copy_url_label: "Copy URL"
-      ical_generation_error_text: "An error occured while generating the iCal URL."
+      ical_generation_error_text: "An error occured while generating the calendar URL."
+      success_message: "The URL \"%{name}\" was successfull copied to your clipboard. Paste it in your calendar client to complete the subscription."
```
Comment (aaron-contreras) [inline config/locales/js-en.yml]:
```suggestion
      success_message: "The URL \"%{name}\" was successfully copied to your clipboard. Paste it in your calendar client to complete the subscription."
```

**Review by aaron-contreras — CHANGES_REQUESTED** [review]
Just some typos. After those are corrected, you're good to go!

File: frontend/src/app/shared/components/modals/get-ical-url-modal/query-get-ical-url.modal.ts (hunk 1 above)
Comment (klaustopher) [reply frontend/src/app/shared/components/modals/get-ical-url-modal/query-get-ical-url.modal.ts]: I just did not do anything else to the file. Will remove

Comment (aaron-contreras) [reply frontend/src/app/shared/components/modals/get-ical-url-modal/query-get-ical-url.modal.ts]: It was fine either way. I know eslint starts complaining the moment you touch a file. Left the commit suggestion there just so you didn't have to go back and do it :)

**Review by aaron-contreras — APPROVED** [review]

**Pull request merged**